dotnet/src/Connectors/Connectors.AI.OpenAI/ChatCompletion/AzureChatCompletion.cs

```diff
@@ -83,7 +82,7 @@ public ChatHistory CreateNewChat(string? instructions = null)
         CompleteRequestSettings requestSettings,
```

Comment:
Should CompleteRequestSettings have the same nullability/default rules as ChatRequestSettings?